exporter/oci/export.go

```diff
@@ -73,12 +74,6 @@ func (e *imageExporter) Resolve(ctx context.Context, opt map[string]string) (exp
 		switch k {
 		case keyImageName:
```

Comment:
@tonistiigi @thaJeztah do we want to support `names` as well as `name` here? (Can be a follow-up PR)